openmetadata-ui/src/main/resources/ui/src/styles/components/table.less

```diff
@@ -9,6 +9,38 @@
   .ant-table-bordered {
     table {
       border: 1px solid #dde3ea;
+
+      tbody {
+        tr {
+          td {
+            border-right: none;
+            background: @white;
+
+            .drag-icon {
+              width: 12px;
+              height: 12px;
+              margin-right: 2px;
+              display: inline-flex;
+              visibility: hidden;
+            }
+
+            .expand-icon {
+              cursor: pointer;
+            }
+
+            .expand-cell-empty-icon-container {
+              width: 16px;
+              height: 16px;
+              display: inline-flex;
+            }
+          }
+        }
+        .ant-table-cell-row-hover {
+          .drag-icon {
+            visibility: initial;
+          }
+        }
+      }
```

Comment:
This will affect all the table, Do we want to update all the tables?

Reply:
@chirag-madlani  remove the unwanted css from td tag